ed_sensor_integration/src/kinect/fitter.cpp

```diff
@@ -458,6 +457,19 @@ void Fitter::checkExpectedBeamThroughEntity(const std::vector<double>& model_ran
 
 // ----------------------------------------------------------------------------------------------------
 
+void Fitter::configureBeamModel(const image_geometry::PinholeCameraModel& caminfo)
+{
+    uint nr_beams = std::min(200, caminfo.fullResolution().width); // don't use more data points than the resolution of your camera
```

Comment:
Why minimal 200 beams?

Reply:
Legacy. previously this was hardcoded to 200.